lib/atmosphere/json_error_handler.rb

```diff
@@ -0,0 +1,16 @@
+module Atmosphere
+  module JsonErrorHandler
+    def render_json_error(msg, options = {})
+      error_json = {
+          message: msg,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.